examples/features/torch_batch_process/batch_inference/compare_with_core_api/README.md

```diff
@@ -0,0 +1,24 @@
+# Batch inference with Core API & Torch Batch Processing API
+## Overview
+This example illustrates how to run distributed batch inference with Core API. Determined's Core API is very flexible
+and can be used to run almost anything, including batch inference. 
+
+With Core API, we are able to write an example that
+- is distributed across worker
+- can be preempted and resumed
+- can be monitored on the Determined UI
+
+However, using Core API directly would require the user to directly handle 
+- low-level parallel programming concepts such as gather, rank 
+- Determined machinery such as creating and loading checkpoint, preemption and resumption
+- initialization of appropriate distributed context
+
+You will see that using the Torch Batch Processing API for the same task is a lot easier as it abstracted away all the 
+low level details and provides useful helper functions.
+## Detailed on this example
+We are running inference with a simple vision model on the MNIST dataset. We then store the prediction outcome to the
```

Comment:
How does one actually get these predictions back? I did some digging and see they are populated to things like `/tmp/179ee6a8-2874-459f-b489-78ce7b402181/rank_0/prediction_output_22` and it seems to imply that I should be able to find these results in the checkpoint, but when I download them, there aren't any predictions there.

Reply:
It depends on how checkpoint is configured on your cluster. For example, a s3 backed cluster, you can just go to the s3 bucket to find the predictions (we print out the path for users).

In this case, it seems like it is using the shared_fs option, it requires the same shared_fs directory is mounted to each agent, then you can find the checkpoint in the shared_fs directory see: https://docs.determined.ai/latest/reference/deploy/config/master-config-reference.html#type-shared-fs